### PR TITLE
Add create of OrderItems when user clicks checkout; closes #120

### LINKED ISSUE #120: As a user, when I click checkout, order items are created for each line item.

### PR DESCRIPTION
**Pull request by EmilyMB**
This works so the order and all associated OrderItems are created when the user clicks "Checkout"

### CODE REVIEW

File: app/controllers/orders_controller.rb [hunk 1]
```diff
@@ -8,7 +8,25 @@ def new
...
-    @order = Order.create(user_id: params[:user_id], status: "ordered")
+    @order = Order.create(user_id:     params[:user_id],
+                          status:      "ordered",
+                          total_price: 0)
+    create_order_items
+    @order.update_attributes(total_price: order_total)
     redirect_to order_path(@order)
   end
+
+  def create_order_items
```
Comment (mikedao) [inline app/controllers/orders_controller.rb:19]: I like the separation into a separate method here.

File: app/models/cart.rb [hunk 2]
```diff
@@ -21,12 +21,4 @@ def count
   def remove_item(item_id)
     cart_items.except!(item_id)
   end
-
-  def total_dollar_amount
-    total = cart_items.each.inject(0) do |sum, (key, count)|
-      item = Item.find(key)
-      sum + item.unit_price * count
-    end
-    "$#{(total / 100)}.00"
```
Comment (mikedao) [inline app/models/cart.rb:30]: Where did this logic go? Where are we converting into currency?

Comment (EmilyMB) [inline app/models/cart.rb:30]: This logic has been moved to the Order model

File: app/controllers/orders_controller.rb (hunk 1 above)
Comment (EmilyMB) [inline app/controllers/orders_controller.rb:19]: :+1:

**Pull request merged**